samples/boards/mec172xevb_assy6906/rom_api/sample.yaml

```diff
@@ -5,8 +5,17 @@ common:
   tags: crypto
   harness: console
   harness_config:
-    type: one_line
-    ordered: false
+    type: multi_line
+    regex:
```

Comment:
@Manimaran-A, why don't you use `ordered:true` ?
It also works and makes the test more specific.